### DISCUSSION TITLE
posts/test/

### DISCUSSION THREAD

**Discussion by giscus** [Parsifa1/blog#1] [Announcements]
# posts/test/

从hexo迁移到Astro框架

https://blog.cloudti.de/posts/test/

**Comment by Parsifa1** [comment]
测试评论

**Comment by Parsifa1** [comment]
markdown test:
# H
## H
### H

**Comment by Parsifa1** [comment]
>自三峡七百里中，两岸连山，略无阙处

--《三峡》郦道元

**Comment by PermeationLoop** [comment]
雖然在羣裏隨便動動嘴皮子就能創造一個新話題，但是真正想讓群友體會的東西 or 想炫耀的東西總是會輕易地淹沒在成千上萬條的：

『你今天的群幼老婆是……』  
『咱貓貓的 xx 用的都是 xx……』  
『/中出 成孕肚！』  
『[貼紙][貼紙][貼紙][貼紙][貼紙][貼紙][貼紙]』  

之中，，，
所以到了某個人生階段，俺反而開始鍾意起『博客 + 留言區』的互動方式，博主你在上面長篇大論，而我在下面針對你的文章發表黑屁，沒有插嘴和 KY，精準而優雅，如入無人之境；雖說評論區仍然有跑題的可能，但是在博客的規模龐大到一定程度之前都不需要擔心這種事情，而且寫留言能更好地加深群友感情不是嗎，，，

補充：下圖是《編程隨想的博客》留言區的過濾方法：

<img src="https://gist.github.com/user-attachments/assets/f3f31b1f-0120-48f4-b46c-bdef6fb9de33" width=598 height=350 alt="已過濾沒有信息量的留言" />

俺最近已決定要多去群友的留言區留言了，沒有博客的群友也可以到 Steam 主頁去．

------

說了這麼多對留言區的態度，俺要開始偏題了，我有三點要說：

### 1. [廣告] 請收藏俺的網站 https://ex-tasty.com 並常來看看

極限風味博客，現已走上週更週記道路，並且評論區開放，歡迎來水．

本博客宗旨：
『『『
  你球的中文獨立博客大多一股程序員味兒，看都看不懂
  **俺要以極限的風味肩負起豐富你球網路文化世界的重任！！**
』』』

少寫晦澀的東西，降低留言區門檻，，，


### 2. 雖然這裏是你的博客，但是留言區是我家客廳

<details>
   <summary> [已摺疊] 🖼️這是一張不應該出現在你博客裏面的色圖，但是我就是要發 </summary>
<br />
  <img src="https://gist.github.com/user-attachments/assets/e8f40218-e403-40ba-afb5-0215a1b62db0" width=907 height=1280 alt="色圖" />
<br />

</details>


### 3. 俺希望你也能幫忙壯大互相利用博客留言區互動的隊伍

🥰👍🏻

**Reply by Parsifa1** [reply]
你是评论区大神👍

**Comment by aurora0x27** [comment]
你好, 我很喜欢你博客的主题, 请问一下是如何在astro paper的基础上改进的? 我能否借鉴你的博客主题?

**Reply by Parsifa1** [reply]
你好 可以的！
倒是没有什么如何改进 我重写了很多东西 我自己都理不清头绪了 我说实话astropaper的css写的很烂 他已经迁移到tailwind 4.0了 我都还没动力迁移...
借鉴请随意！ 我repo应该没有什么敏感资料，如果有什么问题可以 tg 问我！
